content/docs/command-reference/add.md

```diff
@@ -243,3 +243,41 @@ In this case, a DVC-file is generated for each file in the `pics/` directory
 tree. No top-level DVC-file is generated, which is typically less convenient.
 For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
+
+To untrack a file or directory just add
+[patterns](https://git-scm.com/docs/gitignore) (corresponding to the location of
+file or directory) under `.dvcignore` file.
+
+Let's create a sample script to understand this fact.
+
+```dvc
+$ echo test_v1 >> test1.txt
+$ echo test_v1_1 >> test2.txt
+
+$ dvc add test*
+
+100% Add|████████████████████████████████████████████|2/2 [00:04,  2.21s/file]
+
+To track the changes with git, run:
+
+        git add .gitignore test1.txt.dvc test.txt.dvc
```

Comment:
What is the `*` for? Typo perhaps?

Please remove the output from the example, we try to avoid those in general because DVC is in active development and output details tent to change  periodically. Docs are easier to maintain when command example output is omitted. Thanks

Reply:
As I have two files named test1.txt and test2.txt and hence to add both of them i did `dvc add test*` which adds all files whose first part is `test`. Although I could do `dvc add test1.txt test2.txt` but I found this to be more programmatical approach.

>Please remove the output from the example

Yeah sure.

Reply:
> As I have two files named test1.txt and test2.txt

Ok got it, that's correct then, thanks.

> I could do dvc add test1.txt test2.txt

Yeah actually I like this better because it's more explicit. We want to be as simple and obvious as possible, no need for smart shortcuts in examples.